app/tests/unit/components/acronyms.spec.js

```diff
@@ -1,10 +1,61 @@
 const helper = require('../../common/helper');
 const acronyms = require('../../../src/components/acronyms');
+const { acronymService } = require('../../../src/services');
 const utils = require('../../../src/components/utils');
 
 helper.logHelper();
 
 
+describe('getAcronym', () => {
+  const acronymDetailFixture = require('./fixtures/acronymDetail.json');
+
+  // Spy/mock the DB access 'find' method on the acronym table
+  const spy = jest.spyOn(acronymService, 'find');
+  jest.mock('../../../src/services/acronym');
```

Comment:
When you do a jest.spyOn, you don't need to do a jest.mock of the entire module. This line is probably unnecessary.

Reply:
fixed